src/material/form-field/form-field.scss

```diff
@@ -184,7 +184,9 @@ $mat-form-field-default-infix-width: 180px !default;
 .mat-form-field.mat-form-field-invalid {
   .mat-form-field-ripple {
     opacity: 1;
-    transform: scaleX(1);
+    // Note that we transition this to `none`, rather than `scaleX(1)`, because the scaling
```

Comment:
Have you confirmed that this doesn't mess up any transitions or animations on the element? IIRC things don't animate nicely from `none` to something

Reply:
I tried toggling it on/off and it did transition as expected. FWIW, this isn't the only place where we transition to `transform: none`, we do it in other places as a workaround to an issue where text becomes blurry on IE.

Reply:
Ok, maybe transform isn't affected by that quirk, I know things like `width` and `height` are.